.circleci/config.yml

```
@@ -116,11 +117,37 @@ jobs:
               docker tag app:build "${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}:${CIRCLE_TAG}"
               docker push ${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}:${CIRCLE_TAG}
             fi
+      - run:
+          name: Prepare environment variables for OIDC authentication
+          command: |
+            echo 'export GOOGLE_PROJECT_ID="moz-fx-experimenter-prod-6cd5"' >> "$BASH_ENV"
```

This is definitely incorrect copypasta. If this worked to produce the test imag https://console.cloud.google.com/artifacts/docker/moz-fx-glam-prod/us/glam-prod/glam?project=moz-fx-glam-prod that's pretty interesting information and suggests we maybe don't need to set this value.

```suggestion
            echo 'export GOOGLE_PROJECT_ID="moz-fx-glam-prod"' >> "$BASH_ENV"
```

Additionally: some of these variables are available via https://app.circleci.com/settings/organization/github/mozilla/contexts/02806e84-f505-4ad2-b8b1-5eda3c797029?return-to=%2F so you don't need to set them in the project-specific environment variables if you use that context. I guess this is in a bit of a confusing state while you're doing both GCR/GAR so as long as it works :+1: